Source/Core/Core/IOS/Network/KD/Mail/MailCommon.h

```diff
@@ -0,0 +1,70 @@
+// Copyright 2023 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include "Common/CommonTypes.h"
+
+namespace IOS::HLE::NWC24::Mail
+{
+constexpr u32 MAIL_LIST_MAGIC = 0x57635466;  // WcTf
+
+#pragma pack(push, 1)
+struct MailListHeader final
+{
+  u32 magic;    // 'WcTf' 0x57635466
+  u32 version;  // 4 in Wii Menu 4.x
+  u32 number_of_mail;
+  u32 total_entries;
+  u32 total_size_of_messages;
+  u32 filesize;
+  u32 next_entry_id;
+  u32 next_entry_offset;
+  u32 unk2;
```

Comment:
`unk2` but no `unk`/`unk1`? I guess you had an unknown value earlier but figured it out, so probably update this. Same for `unk3` below.